internal/cluster/validator.go

```diff
@@ -528,23 +527,85 @@ func (v *clusterValidator) printNtpServerConfigured(c *clusterPreprocessContext,
 	}
 }
 
-func (v *clusterValidator) isOcsRequirementsSatisfied(c *clusterPreprocessContext) validationStatus {
-	validationStatus := (v.operatorsManager.ValidateOCSRequirements(c.cluster))
-	if validationStatus == "pending" {
-		return ValidationPending
-	} else if validationStatus == "success" {
-		return ValidationSuccess
+// sufficientMemoryAvailableForOperators: A cluster is considered to have a sufficient amount of memory when the amount of memory usable per node is greater
+// than the sum of the minimum memory requirements for each of the operators enabled for the cluster.
+// In a nutshell, a successful validation looks like this:
+//     sum(Node usable memory) > sum (minimum memory required for each enabled operator)
+func (v *clusterValidator) sufficientMemoryAvailableForOperators(c *clusterPreprocessContext) validationStatus {
+
+	opMgr := operators.NewManager(v.log)
+	if !opMgr.AnyOperatorEnabled(c.cluster) {
+		return boolValue(true)
+	}
+
+	// gather the total memory usable available in the cluster
```

Comment:
what do you think about extracting total cluster memory collection to a method?

Reply:
I'll take a look at the code to see if this logic is being used elsewhere. Thanks for the suggestion :smiley:

Reply:
Done: moved to its own struct method